ci/docker/python-wheel-windows-vs2019.dockerfile

```diff
@@ -84,7 +84,7 @@ RUN (if "%python%"=="3.9" setx PYTHON_VERSION "3.9.13" && setx PATH "%PATH%;C:\P
     (if "%python%"=="3.10" setx PYTHON_VERSION "3.10.11" && setx PATH "%PATH%;C:\Python310;C:\Python310\Scripts") & \
     (if "%python%"=="3.11" setx PYTHON_VERSION "3.11.9" && setx PATH "%PATH%;C:\Python311;C:\Python311\Scripts") & \
     (if "%python%"=="3.12" setx PYTHON_VERSION "3.12.5" && setx PATH "%PATH%;C:\Python312;C:\Python312\Scripts") & \
-    (if "%python%"=="3.13" setx PYTHON_VERSION "3.13.0-rc1" && setx PATH "%PATH%;C:\Python313;C:\Python313\Scripts")
+    (if "%python%"=="3.13" setx PYTHON_VERSION "3.13.0" && setx PATH "%PATH%;C:\Python313;C:\Python313\Scripts")
```

Comment:
`3.13.0` is still not on chocolatey, see: https://community.chocolatey.org/packages/python313
That's why it fails